.gitignore

```diff
@@ -187,6 +187,9 @@ node_modules/
 /ajax/libs/react-data-grid/*-dm*/
 /ajax/libs/jquery-easing/1.3.0/
 /ajax/libs/jquery-easing/1.3.1/
+/ajax/libs/teadao/0.1.0/
+/ajax/libs/teadao/0.4.0/
+/ajax/libs/teadao/0.5.0/
```

Comment:
There is no the auto-update set `library` basepath in v0.4.0/0.5.0, so why do we need to set them in the ignore list? It'll fetch nothing be default.